manifests/base/postgres/postgres.yaml

```diff
@@ -230,7 +230,8 @@ spec:
               command:
                 - /bin/sh
                 - -c
-                - exec pg_isready -U "$(POSTGRES_USER)" -d "dbname=$(POSTGRES_DATABASE)" -h 127.0.0.1 -p 5432
+                - |
+                  exec pg_isready -U "postgres" - -d "dbname=$(POSTGRES_DATABASE)" -h 127.0.0.1 -p 5432
```

Comment:
Is postgres always the admin user and need that for the checking?

Reply:
For the readiness checks yeah it has to be the admin user which seems odd but :shrug: